tsconfig.json

```diff
@@ -13,8 +13,13 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"strict": true,
-		"target": "ESNext"
+		"target": "ESNext",
+		"plugins": [
+			{
+				"name": "next"
+			}
```

Comment:
[Question] Ooh I haven't seen this before and can't find docs on it. Is this described anywhere?

I see that `next build` and `next dev` add it automatically, so I guess we need it 😄

Reply:
yes, this was added automagically. I have no idea 😆 tbh